safe_cli/operators/safe_operator.py

```diff
@@ -782,7 +785,13 @@ def batch_safe_txs(self, safe_nonce: int, safe_txs: [SafeTx]) -> SafeTx:
                     )
                 )
 
-        if len(multisend_txs) > 1:
+        if len(multisend_txs) == 1:
+            safe_tx.safe_tx_gas = 0
+            safe_tx.base_gas = 0
+            safe_tx.gas_price = 0
+            safe_tx.signatures = b""
+            safe_tx.safe_nonce = safe_nonce  # Resend single transaction
+        elif multisend:
```

Comment:
Was we using multisend for one tx?

Reply:
Well, this was implemented and I don't want to touch that logic